### PR TITLE
add warning page

### PR DESCRIPTION
**Pull request by thejeremias**
Here's my PR. Check it for me.

### CODE REVIEW

File: blockpage.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ function fetchProtectionRules(url,status){
         addRules: domains.map((domain, index) => ({
           id: index + 1,
           priority: 1,
-          action: { type: "redirect", redirect: { url: "https://google.com/" } },
+          action: { type: "redirect", redirect: { url: chrome.runtime.getURL("warning_page.html") } },
```
Comment (immattdavison) [inline blockpage.js:59]: Can we change this so that it memorises the page they were on and redirects them to that page when they pause NMD?

**Review by immattdavison — CHANGES_REQUESTED** [review]

File: warning_page_script.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    disableNMD();
+    setTimeout(() => {
+        enableNMD();
+        PAUSE_BUTTON.setAttribute("disabled", false);
+        PAUSE_BUTTON.setAttribute("style", "opacity: 100%");
+    }, PAUSE_MINUTES*60*1000);
+    PAUSE_BUTTON.setAttribute("disabled", true);
+}
+
+PAUSE_BUTTON.addEventListener("click", pause);
```
Comment (immattdavison) [inline warning_page_script.js:46]: pause seems to turn off NMD indefinitely, we need this to turn it off and automatically turn it back on 15-30 mins later

**Review by immattdavison — CHANGES_REQUESTED** [review]

File: warning_page_script.js (hunk 2 above)
Comment (thejeremias) [reply warning_page_script.js:46]: Did you see the setTimeout()?

File: blockpage.js (hunk 1 above)
Comment (thejeremias) [reply blockpage.js:59]: Sounds good.

**Comment by thejeremias** [comment]
> Hey @thejeremias would it be possible to have this page pop up over the top of the page rather than have a complete redirect? like a modal in the middle and you can't access the page until NoMoreDomains is powered off either using a button on the modal or in the browser extension?

I don't know how to make that.

**Pull request closed without merging**